src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -477,6 +477,12 @@
 
 "ExposureSubmissionError_defaultError" = "Allgemeiner Exposure Submission Fehler";
 
+"ExposureSubmissionError_internal" = "Fehler 11. Es ist ein interner Fehler passiert. Bitte starten Sie das Telefon neu und probieren Sie es morgen erneut.";
```

Comment:
Can you add an intermediate english translation for those and put these also into turkish language file?;)